Kickstarter-iOS/Views/Controllers/CommentRepliesViewController.swift

```diff
@@ -54,6 +89,20 @@ final class CommentRepliesViewController: UITableViewController {
 
         self?.tableView.reloadData()
       }
+
+    self.viewModel.outputs.configureCommentComposerViewWithData
+      .observeForUI()
+      .observeValues { [weak self] data in
+        self?.commentComposer.configure(with: data)
+      }
+  }
+}
+
+// MARK: - CommentComposerViewDelegate
+
+extension CommentRepliesViewController: CommentComposerViewDelegate {
+  func commentComposerView(_: CommentComposerView, didSubmitText _: String) {
+    self.commentComposer.resetInput()
```

Comment:
We'll likely want to drive this from the VM in the future, if you're not doing that as a part of this PR be sure to add a `TODO` here.